superdesk/io/feed_parsers/__init__.py

```diff
@@ -63,13 +63,17 @@ def set_dateline(self, item, city=None, text=None):
         :type text: str
         """
 
-        item['dateline'] = {}
+        if 'dateline' not in item:
+            item['dateline'] = {}
```

Comment:
For cases like this, the `setdefault()` method comes handy, avoiding the need for the explicit check :wink:
```py
item.setdefault('dateline', {})
```
Cheers!